.github/workflows/ci.yml

```diff
@@ -0,0 +1,64 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - '**'
+    tags:
+      - 'v*'
+  pull_request:
```

Comment:
Why do we need it? Branches + pull_request creates duplicate builds

Reply:
The only reason when someone forks repo, makes changes and provides pull request from forked repo, the only pull request hook will trigger.

Reply:
https://img.namecheap.net/bpfhG2f98AvTUxV5GsrCmY.png